fastn.com/backend/wasm.ftd

```diff
@@ -0,0 +1,42 @@
+-- ds.page: WASM backends with [ft-sdk](https://github.com/fastn-stack/ft-sdk/)
+
+[ft-sdk](https://github.com/fastn-stack/ft-sdk/) is a rust crate that can be
```

Comment:
We prefer `ft-sdk` instead of just "ft-sdk" (for crate names). Also it should be Rust, not rust.

Reply:
https://github.com/fastn-stack/fastn/pull/2056/commits/ffcdcfe2c1d2bb2134ac37150424cb5ee4daeb24 done in